### PR TITLE
fix: tolerate space when parsing file directory

### PR DESCRIPTION
**Pull request by jazelly**
~~Partially fix: https://github.com/clinicjs/node-clinic/issues/448~~

Fixes: https://github.com/clinicjs/node-clinic/issues/448

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ function execute (instance, args, env, nodeOptions, cb) {
     */
     process.once('SIGINT', noop)
 
-    instance.process = spawn(args[0], args.slice(1), { stdio: ['ignore', 'inherit', 'inherit'], env })
+    instance.process = spawn(args[0], [args.slice(1)], { stdio: ['ignore', 'inherit', 'inherit'], env })
```
Comment (Eomm) [inline src/index.js:35]: Slice returns an array already

**Review by Eomm — COMMENTED** [review]
Thanks for looking into it!

I think the project needs a new unit test for this fix

**Comment by jazelly** [comment]
Hi @Eomm thanks for the review.

I tried adding unit test to this but it's not straight forward.

The best I can think of is to test `chile_process.spawn()`  to see if it accepts double quotes, but it obeys the point of unit testing, i.e. it it not testing `tool.collect()` in this project but other outside modules.

**Review by RafaelGSS — COMMENTED** [review]
It's better to do it directly on `.spawn`. Also, how does it partially fix the issue? what's missing?

**Comment by jazelly** [comment]
@RafaelGSS 

> It's better to do it directly on .spawn
Thanks. I'll add the unit test.

>  what's missing?
Since this PR only aims fixing the `space` issue, but looks like the `ECONNREFUSED` is not an issue of this repo.

Also, windows stripped out `\` in the directory. Will fix that as well.

**Comment by jazelly** [comment]
I messed up with my email config in previous commits. Had to squashed the commits. 

I understand that might cause some difficulty to review the changes, but please have another look. CI should be happy now.

File: test/basic.test.js [hunk 2]
```diff
@@ -73,6 +74,27 @@ test('cmd - test collect - data exists, html generated', t => {
...
+  }
+
+  this.process = spawn(
+    process.execPath,
+    [path.join('test', 'fixtures', 'randomHashes.js')],
+    { stdio: ['ignore', 'inherit', 'inherit'], env }
+  )
+
+  this.process.once('exit', (code) => {
+    t.notOk(code)
```
Comment (Eomm) [inline test/basic.test.js:93]:
```suggestion
    t.equal(code, 0)
```

**Review by Eomm — APPROVED** [review]
I have no power here, but LGTM!
Thks!

**Comment by jazelly** [comment]
@RafaelGSS Do you think this can be merged?

**Review by RafaelGSS — APPROVED** [review]

**Pull request merged**